altair/examples/line_chart_with_custom_legend.py

```diff
@@ -0,0 +1,37 @@
+"""
+Line Chart with Custom Legend
+-----------------------------
+This example uses the argmax aggregation function in order to create a custom
+legend for a line chart.
+"""
+# category: line charts
+import altair as alt
+from vega_datasets import data
+
+
+source = data.stocks()
+
+base = (
+    alt.Chart(source)
+    .transform_filter("datum.symbol !== 'IBM'")
+    .encode(color=alt.Color("symbol", legend=None))
+    .properties(width=500)
+)
+
+line = base.mark_line().encode(x="date", y="price")
+
+
+last_price = (
+    base.mark_circle()
+    .transform_aggregate(last_date="argmax(date)", groupby=["symbol"])
+    .encode(x=alt.X("last_date['date']:T"), y=alt.Y("last_date['price']:Q"))
+)
```

Comment:
```suggestion
llast_price = base.mark_circle().encode(
    x=alt.X("last_date['date']:T"),
    y=alt.Y("last_date['price']:Q")
).transform_aggregate(
    last_date="argmax(date)",
    groupby=["symbol"]
)
```

Reply:
There's a typo in line 24: `llast_price` instead of `last_price`. I don't know how to edit a suggestion (I don't even know if it's possible to do that), so I won't commit this for the moment.

Reply:
Great catch! I fixed it and committed.